runtime/tests/tests.rs

```diff
@@ -151,16 +152,34 @@ fn start_era(era_index: EraIndex) {
 	assert_eq!(Staking::current_era(), era_index);
 }
 
-pub fn current_total_payout_for_duration(duration: u64) -> Balance {
+fn current_total_payout(validator_count: Balance) -> Balance {
+	let now = Timestamp::now();
+	let previous_era_start = <crml_staking::CurrentEraStart<Runtime>>::get();
+	let era_duration = now - previous_era_start;
+
 	crml_staking::inflation::compute_total_payout(
 		<Runtime as crml_staking::Trait>::RewardCurve::get(),
-		<Staking>::slot_stake() * 2,
+		<crml_staking::SlotStake<Runtime>>::get() * validator_count,
 		GenericAsset::total_issuance(&CENNZ_ASSET_ID),
-		duration,
+		era_duration.saturated_into::<u64>(),
 	)
 	.0
 }
 
+fn reward_validators(validators: &[(AccountId, AccountId)]) -> Balance {
+	let validator_len = validators.len() as Balance;
+	let total_payout = current_total_payout(validator_len);
+	assert!(total_payout > 1);
+	Staking::reward_by_ids(
+		validators
+			.iter()
+			.map(|v| (v.0.clone(), 1))
+			.collect::<Vec<(AccountId, u32)>>(),
+	);
```

Comment:
Can we leave a comment to explain what this `reward_by_ids` is doing?
[Optional] can refactor into something like:
```rust
// Mapping of stash account and points to be assigned to them
let validators_points: :<Vec<(AccountId, u32)>> = validators
    .iter()
    .map(|v| (v.0.clone(), 1))
    .collect();
Staking::reward_by_ids(validators_points); // CurrentEraPointsEarned storage is updated with new points
```